.gitignore

```diff
@@ -1,15 +1,32 @@
-# ignore visual studio directies and local files
-.vscode
```

Comment:
Could you add the `.vscode` line back?  The Visual Studio Code is different than Visual Studio 2019.

Reply:
Added back in 66fa949